spec/model/user_spec.rb

```diff
@@ -0,0 +1,43 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  subject { User.create(name: 'Harry Potter', photo: 'http://tactes.com/image', bio: 'User bio', posts_counter: 0) }
+
+  before { subject.save }
+
+  describe 'association' do
+    it 'has many posts' do
+      have_many(:posts)
+    end
+    it 'has many comments' do
+      have_many(:comments)
+    end
+    it 'has many likes' do
+      have_many(:likes)
+    end
+    # it { should have_many(:comments) }
+    # it { should have_many(:likes) }
```

Comment:
Optional

- It'd be nicer if you removed the unused code for a cleaner file.

Reply:
Thank you @MahmoudBakr23